scripts/globals/magic.lua

```diff
@@ -582,6 +587,11 @@ function getSpellBonusAcc(caster, target, spell, params)
         end
     end
 
+    --add for blm elemental magic merits
+    if skill == dsp.skill.ELEMENTAL_MAGIC and caster:getMainJob() == dsp.job.BLM and caster:getMainLvl() >= 75 then
```

Comment:
Remove main job and main level check. They are not needed. getMerit handles this.